### DISCUSSION TITLE
Is kbatch working for anyone?

### DISCUSSION THREAD

**Discussion by jessjaco** [microsoft/PlanetaryComputer#333] [General]
I can't get kbatch to do anything, all I get are "not found" or "method not allowed" errors.  Is it working for anyone else? 

```bash
> kbatch job submit --name=list-files --command='["ls", "-lh"]' --image=alpine
[15:30:44] INFO     HTTP Request: POST https://pccompute.westeurope.cloudapp.azure.com/compute/services/kbatch/jobs/ "HTTP/1.1 405 Method Not Allowed" 
...more stuff...
raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 4 column 1 (char 3)
```

**Comment by jessjaco** [comment]
Oh I guess it's being removed https://github.com/microsoft/PlanetaryComputerDataCatalog/commit/8e5b2d178d0a7fbe9a2b43eeb54dda6e522106a1

**Comment by TomAugspurger** [comment]
Sorry for the delay in responding. Yeah, we did recently remove k-batch from the Hub. I reached out to people who had used it recently, but might have missed you, apologies.

**Comment by DavidCarMor** [comment]
Is there another way to submit asynchronous jobs?

**Reply by TomAugspurger** [reply]
Not on the PC Hub, no.

https://learn.microsoft.com/en-us/azure/architecture/guide/technology-choices/compute-decision-tree has a list of other compute services on Azure that might work better for you. They'll have equally fast access to the data as the Hub.
